app/javascript/components/payment-error-button.vue

```diff
@@ -1,17 +1,18 @@
 <template>
   <div class="payment-error-button">
     <modal name="apology-modal">
+      <div class="invoice__apology-modal-flexbox">
       <div class="invoice__apology-modal-text">
-        <div v-if="showApologyText">
-          <h1> Error en el pago </h1>
+        <div class="invoice__apology-modal-text-first" v-if="showApologyText">
+          <h5> Error en el pago </h5>
           Disculpe las molestias :(, lo vamos a revisar para que puedas pagar en el futuro
         </div>
-        <div v-if="showCartOnDebtModal">
-          <h1> Productos </h1>
+        <div class="invoice__apology-modal-text-second" v-if="showCartOnDebtModal">
```

Comment:
Expected indentation of 10 spaces but found 8 spaces   vue/html-indent<br>Attribute "if" should be on a new line                 vue/max-attributes-per-line